docker-compose.yaml

```diff
@@ -12,13 +12,13 @@ services:
         ipv4_address: ${U7S_NODE_IP}
     ports:
       # etcd
-      - 2379:2379
+      - 2379:${U7S_ETCD_EXPOSED_PORT}
```

Comment:
How was this tested?
The ordering seems opposite.

Reply:
You're right - they are flipped! I set them to the same thing so often I didn't realize it wasn't `<container>:<host>`